css/style.css

```diff
@@ -1,19 +1,19 @@
-/* 
 
-Style Hints: 
+Body{
```

Comment:
I would make it a habit to not capitalize your CSS selectors or property names. Things like ids and classes are case-sensitive, so it's good to be consistent.